AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -793,136 +835,16 @@ msgstr ""
 "\n"
 " Dengue Situation - %(num_mun)s Monitored cities \n"
 
-#: dados/templates/data_public_services.html:19
-msgid "Serviços de Dados Públicos"
-msgstr "Public Data Service"
-
-#: dados/templates/data_public_services_api.html:46
-#: dados/templates/data_public_services_maps.html:69
-#: dados/templates/data_public_services_maps_doc.html:47 templates/base.html:74
-msgid "Dados"
-msgstr "Data"
-
-#: dados/templates/data_public_services_api.html:63
-msgid "Tabela de conteúdo"
-msgstr "Table of content"
-
-#: dados/templates/data_public_services_maps.html:69
-msgid "Mapas"
-msgstr "Maps"
-
-#: dados/templates/data_public_services_maps.html:79
-msgid "Para mais informações, acesse a documentação do serviço de mapas."
-msgstr "For more information, access the map service documentation"
-
-#: dados/templates/data_public_services_maps.html:84
-msgid "Filtro Código IBGE"
-msgstr "IBGE code filter"
-
-#: dados/templates/data_public_services_maps.html:88
-msgid "Digite aqui para filtrar as opções do código IBGE."
-msgstr "Type here to filter IBGE code options."
-
-#: dados/templates/data_public_services_maps.html:93
-msgid ""
-"Digite o código IBGE ou parte dele para buscar o código IBGE disponível."
-msgstr "Type the IBGE code or fragment to search available IBGE code."
-
-#: dados/templates/data_public_services_maps.html:98
-msgid "Código IBGE"
-msgstr "IBGE code"
-
-#: dados/templates/data_public_services_maps.html:103
-#: dados/templates/state_cities.html:198
-msgid "Selecione municipio(s)"
-msgstr "Select one or more cities"
-
-#: dados/templates/data_public_services_maps.html:108
-msgid "Selecione o código IBGE da cidade desejada"
-msgstr "Select the IBGE code of the desired city"
-
-#: dados/templates/data_public_services_maps.html:115
-#: dados/templates/report_filter_city.html:79
-msgid "Selecione o município desejado para a consulta"
-msgstr "Select the desired city to search"
-
-#: dados/templates/data_public_services_maps.html:119
-msgid "Mapa"
-msgstr "Map"
+#: dados/templates/details.html:20
+msgid "Alerta Dengue Rio"
+msgstr "Alerta Dengue Rio"
```

Comment:
maybe "Dengue Alert Rio de Janeiro"

Reply:
This is out of context: but a more up-to-date translation would be "Infodengue Rio de Janeiro" if this refers to the Project's name.

Reply:
ok